azurelinuxagent/ga/exthandlers.py

```diff
@@ -764,17 +766,17 @@ def version_ne(self, other):
         return FlexibleVersion(self_version) != FlexibleVersion(other_version)
 
     def get_installed_ext_handler(self):
-        lastest_version = self.get_installed_version()
```

Comment:
Wow in all this time never saw this typo! Thanks for fixing it!

Reply:
Me neither until I had to closely read this part of the code :D

Reply:
Oy, and me thinking "lastest" was the "last of the latest" (as in "bestest")!